core/src/components/datetime/test/minmax/datetime.e2e.ts

```diff
@@ -301,21 +301,20 @@ configs({ directions: ['ltr'], modes: ['ios'] }).forEach(({ title, config }) =>
           <ion-datetime
             min="2022-01-15"
             value="2022-02-01"
-            presentation="date"
+            presentation="month-year"
           ></ion-datetime>
         `,
           config
         );
 
         const datetime = page.locator('ion-datetime');
-        const monthYearToggle = page.locator('ion-datetime .calendar-month-year');
         const monthColumnItems = page.locator('ion-datetime .month-column .picker-item:not(.picker-item-empty)');
+        const ionChange = await page.spyOnEvent('ionChange');
```

Comment:
This test started to flaky after my changes. This test touches the wheel picker which does not use any of the changed code. I noticed that the test was not waiting for `.datetime-ready` and it was not waiting for the `ionChange` event, so it's possible that my change made datetime slightly faster such that this test is now consistently failing.